src/borg/helpers.py

```diff
@@ -1696,3 +1696,26 @@ def write(self, s):
                 except OSError:
                     pass
         return len(s)
+
+
+def swidth_slice(string, desired_length):
+    """
+    Return a slice of *desired_length* from *string*.
+
+    Negative *desired_length* means from the end of string.
```

Comment:
Good point. Renamed desired_length -> max_width which then implies that it's a maximum that may not be fully filled, and that it's not about string length. Added explanation of cells/columns.